### DISCUSSION TITLE
Time tracking favourite section.

### DISCUSSION THREAD

**Discussion by mindaugaswashere** [runelite/runelite#18303] [Ideas]
![image](https://github.com/user-attachments/assets/e4b55b68-b6dd-4f04-9e84-6808f3720d4b)
Would be nice improvement to have "Favourite" section where I can add my own time trackings in one window, instead of navigating trough many ones, and in some cases even scroll down and search for it, instead of having all of my marked timers in one place. 👍

**Comment by cdfisher** [comment]
This is effectively a duplicate of #16268

**Reply by mindaugaswashere** [reply]
Its 2 year old ticket, does anyone even reads them?
